src/agenda/reservation-list/style.ts

```diff
@@ -37,6 +37,6 @@ export default function styleConstructor(theme: Theme = {}) {
     indicator: {
       marginTop: 80
     },
-    ...(theme['stylesheet.agenda.list'] || {})
```

Comment:
removing these is a breaking change. 
You can only add the optional chaining to allow passing nested object but we must keep the old one as it will stop working for users who still use this pattern.